.gitlab-ci.yml

```diff
@@ -72,5 +77,18 @@ release:
     - make deps
     - poetry config pypi-token.pypi $PYPI_PASSWORD
     - make publish
+
+
+release-github:
```

Comment:
I think if you split github release, you'll need to make it a dependency on pypi release and consume pypi release artifacts inside github release job.

Reply:
https://docs.gitlab.com/ee/ci/yaml/#needsartifacts

So this sep will require roughly:

```
needs:
  - job: release-pypi
     artifacts: true
```

Then all artifacts generated by pypi build will be available to gh job.

Reply:
Updated.